src/components/molecules/Visualizer/Engine/Cesium/common.ts

```diff
@@ -633,6 +634,29 @@ export async function flyToGround(
   );
 }
 
+export function getPixelDistance(scene: Scene, verticalPosition = "bottom") {
+  const width = scene.canvas.clientWidth;
+  const height = scene.canvas.clientHeight;
+
+  const sampleHeight =
```

Comment:
I checked briefly cesium docs and the getPickRay's second param is where we are using `sampleHeight` is `result`, which it explains is the object that we store the result. Which makes me alittle confused why we are passing a number in to there.
Can you explain that a little more? And also think of a better name than sampleHeight?

Reply:
Actually `sampleHeight` is the second param of `new Cartesian2` not `getPickRay`. The reason why i want to use the word `sample` is that in most cases our camera is a perspective camera, which will have different pixel distance in different point and we need to pick one point as a sample. Here the `sampleHeight` actually means the sample point's height (or top better?) on screen space. Emmm maybe need to change widht/height to left/top?